### ISSUE TITLE
Overfitting after 1 epoch on MSR-VTT

### ISSUE THREAD

**Issue by JacobKong** [ArrowLuo/CLIP4Clip#56]
Hi:

I ran the CLIP4Clip with ViT-B/32 and ViT-B/16 on MSR-VTT dataset. After adding the log to tensorboard, it seems the model is overfitting adter 1 or 2 epoch on MSR-VTT, cause the value of recall@K and mean Rank is getting higher while the training loss is decrease.

So do you have any idea of this problem?

![image](https://user-images.githubusercontent.com/10401849/149702159-53d76705-797e-4e26-b650-74912fbab2af.png)

Thanks

**Comment by ArrowLuo** [comment]
Hi @JacobKong, maybe you can test with a smaller lr. Or larger dataset will make the training stable.

**Comment by JingXiaolun** [comment]
@JacobKong , I met similar problem. I have been confused  with this problem for a long time, can you show me your solution? Looking forward to your reply
